packages/jto/src/client/index.css

```
@@ -1,11 +1,181 @@
+}
+
+/* Token → Tailwind color mapping. `inline` so each utility carries the
+   `hsl(var(--x))` expression and re-resolves under `.dark` wherever the class
+   sits, rather than being frozen at :root. */
+@theme inline {
+  --font-sans:
+    'Inter Variable', 'Inter', system-ui, -apple-system, 'Segoe UI', sans-serif;
+  /* Matches what Monaco renders in the editor pane, so inline code chips and
+     the editor read as the same typeface. */
+  --font-mono:
+    ui-monospace, SFMono-Regular, 'SF Mono', Menlo, Consolas, monospace;
```

Stylelint (17.14.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)